modules/persistent-storage-local-discovery.adoc

```diff
@@ -0,0 +1,160 @@
+
+* You have attached local disks to {product-title} nodes.
+
+* You have access to the {product-title} web console and the `oc` command-line interface (CLI).
+
+.Procedure
+
+. To enable automatic discovery of local devices from the web console:
+
+.. In the _Administrator_ perspective, navigate to *Operators* -> *Installed Operators* and click on the *Local Volume Discovery* tab.
+
+.. Click *Create Local Volume Discovery*.
+
+.. Select either *All nodes* or *Select nodes*, depending on whether you want to discover available disks on all or specific nodes.
++
+[NOTE]
+====
+Only worker nodes are available, regardless of whether you filter using *All nodes* or *Select nodes*.
+====
++
+.. Click *Create*.
+
+A local volume discovery instance named `auto-discover-devices` is displayed.
+
+. To display a continuous list of available devices on a node:
+
+.. Log in to the {product-title} web console.
+
+.. Navigate to *Compute* -> *Nodes*.
+
+.. Click the node name that you want to open. The "Node Details" page is displayed.
+
+.. Select the *Disks* tab to display the list of the selected devices.
++
+The device list updates continuously as local disks are added or removed. You can filter the devices by name, status, type, model, capacity, and mode.
+
+. To automatically provision local volumes for the discovered devices from the web console:
+
+.. Navigate to *Operators* -> *Installed Operators* and select *Local Storage* from the list of Operators.
+
+.. Select *Local Volume Set* -> *Create Local Volume Set*.
+
+.. Enter a volume set name and a storage class name.
+
+.. Choose *All nodes* or *Select nodes* to apply filters accordingly.
++
+[NOTE]
+====
+Only worker nodes are available, regardless of whether you filter using *All nodes* or *Select nodes*.
+====
++
+.. Select the disk type, mode, size, and limit you want to apply to the local volume set, and click *Create*.
++
+A message displays after several minutes, indicating that the "Operator reconciled successfully."
+
+[start=3]
+. Alternatively, to provision local volumes for the discovered devices from the CLI:
+
+.. Create an object YAML file to define the local volume set, such as `local-volume-set.yaml`, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: local.storage.openshift.io/v1alpha1
+kind: LocalVolumeSet
+metadata:
+  name: example-autodetect
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+      - matchExpressions:
+          - key: kubernetes.io/hostname
+            operator: In
+            values:
+              - worker-0
+              - worker-1
+  storageClassName: example-storageclass <1>
+  volumeMode: Filesystem
+  fstype: ext4
+  maxDeviceCount: 10
+  deviceInclusionSpec:
+    deviceTypes: <2>
+      - disk
+      - part
+    deviceMechanicalProperty:
+      - NonRotational
+    minSize: 10G
+    maxSize: 100G
+    models:
+      - SAMSUNG
+      - Crucial_CT525MX3
+    vendors:
+      - ATA
+      - ST2000LM
+status:
```

Comment:
If instructing users to create YAML - I would personally skip status from such objects because users don't have to specify them.

Reply:
Removed `status` field.